frontend/src/services/licenceService.js

```diff
@@ -0,0 +1,15 @@
+import ApiService from '@/common/apiService'
+import { ApiRoutes } from '@/utils/constants'
+
+export default {
+  async getLicenceDetails(licenceId) {
+    try {
+      if (!licenceId) return
+      const response = await ApiService.apiAxios.get(`${ApiRoutes.LICENCES}/${licenceId}/licence-details`)
```

Comment:
See my note in backend about licence-details url.

Reply:
please see my above comment